src/calculator.js

```diff
@@ -1,48 +1,26 @@
-exports._check = () => {
-  // DRY up the codebase with this function
-  // First, move the duplicate error checking code here
-  // Then, invoke this function inside each of the others
-  // HINT: you can invoke this function with exports._check()
-};
-
-exports.add = (x, y) => {
+exports._check = (x, y) => {
   if (typeof x !== 'number') {
     throw new TypeError(`${x} is not a number`);
   }
   if (typeof y !== 'number') {
     throw new TypeError(`${y} is not a number`);
   }
-  return x + y;
+};
+
+exports.add = (x, y) => {
+  return exports._check(x, y) || (x + y);
 };
 
 exports.subtract = (x, y) => {
-  if (typeof x !== 'number') {
-    throw new TypeError(`${x} is not a number`);
-  }
-  if (typeof y !== 'number') {
-    throw new TypeError(`${y} is not a number`);
-  }
-  return x - y;
+  return exports._check(x, y) || (x - y);
 };
 
 exports.multiply = (x, y) => {
```

Comment:
Unexpected block statement surrounding arrow body.

###### See http://eslint.org/docs/rules/arrow-body-style for details about this rule